libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/TabsView/utils.ts

```diff
@@ -41,8 +41,17 @@ export function getInfo(
       body = localization.Interpret.DatasetExplorer.helperText;
       title = localization.Interpret.DatasetExplorer.infoTitle;
     } else if (dataAnalysisOption === DataAnalysisTabOptions.TableView) {
-      body = localization.ModelAssessment.FeatureImportances.IndividualFeature;
       title = localization.ModelAssessment.FeatureImportances.InfoTitle;
+      let hasTextImportances = false;
+      if (props && props.modelExplanationData) {

```

Comment:
nit: curious if props is really needed in if condition. Does just if(props.modelEx...) works?

Reply:
good catch, changed to use props?.modelExplanationData instead